### PR TITLE
add hackathon 2017 blog post trip reports

### PR DESCRIPTION
**Pull request by avsm**
- content of post assembled by @gemmag
- photos to be added in a follow up commit

### CODE REVIEW

File: tmpl/blog/2017-march-hackathon-roundup.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+----
+
+## Thomas Leonard
+
+This was my second time at the hackathon, and it was great to see everyone and work on Mirage stuff again! I brought along a NUC which provided an additional wireless access point, running a Mirage/Xen DHCP server using haesbaert's [charrua](https://github.com/mirage/charrua-core library - one of the fruits of last year's efforts.
```
Comment (yomimono) [inline tmpl/blog/2017-march-hackathon-roundup.md]: need a `)` after `charrua-core`

**Comment by yomimono** [comment]
Thanks for putting this together, @GemmaG and @avsm! :)

File: tmpl/blog/2017-march-hackathon-roundup.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+Numerous social contacts, political discussions, technical challenges
+were discussed in smaller and bigger groups. Lots of pull requests were
+opened and merged - we kept the DSL line busy with git pushes and pulls
+:) - sometimes overly busy.
+
+In contrast to [last year](/blog/2016-spring-hackathon), we organised several events:
+
+- Body self-awareness workshop (by the resident dancers)
+- Hiking to waterfalls on Sunday
+- Screening of [Hidden Figures](http://www.imdb.com/title/tt4846340/) on Monday
```
Comment (amirmc) [inline tmpl/blog/2017-march-hackathon-roundup.md]: May not want to mention this... ;)

Comment (avsm) [inline tmpl/blog/2017-march-hackathon-roundup.md]: hahaha, good point

**Pull request merged**